frontend/src/components/sidebar/card.jsx

```diff
@@ -71,7 +95,7 @@ class Card extends React.Component {
                 <div className="card-header-container">
                     <FontAwesomeIcon className="fa-minus" icon={faMinus} />
                   <div className={"card-head-active"}>
-                    0.1 Miles Away
+                    {`${this.state.distance} miles away`}
```

Comment:
We should eventually do something here incase distance is null so it doesnt just say 'miles away'

Reply:
yes i agree i try to break it later and see what i can do to get around something like that